src/components/reusable/dropdown/dropdown.ts

```diff
@@ -341,6 +384,11 @@ export class Dropdown extends FormMixin(LitElement) {
         ${this._isInvalid
           ? html`
               <div class="error">
+                <kd-icon
+                  class="error-info-icon"
+                  aria-label=${this._textStrings.error}
```

Comment:
Can also add a `title` attribute here.

Reply:
Added  :    title=${this._textStrings.error}